### ISSUE TITLE
GET DATE format does not seem to match format specified in &H2B for certain machines

### ISSUE THREAD

**Issue by FiXato** [ppeccin/WebMSX#75]
For the (I guess non-standard) MSX2+ Europe and America machines the date format returned from `get date a$: print a$` does not seem to match the format specified in bits 6,5,4 in `&H2B` (see http://map.grauw.nl/resources/msxsystemvars.php for details):

Example using MSX2+ America machine:
![image](https://user-images.githubusercontent.com/4342/120245136-8b0be080-c26c-11eb-819b-f06b7804e953.png)
https://msxpen.com/codes/-Mb3Qw8ovBQFirYVt-lp
Returned date (21/06/01) is in Japanese Y/M/D format (probably because the system is based on a Japanese MSX2+), while the format says it should be format 1, M/D/Y (06/01/21)

Example using MSX2+ Europa machine:
![image](https://user-images.githubusercontent.com/4342/120245157-9c54ed00-c26c-11eb-9fb3-a0b0d12e3d78.png)
https://msxpen.com/codes/-Mb3ROWqhsvDxr5DIZOY
Again, returned date (21/06/01) is in Japanese Y/M/D format (probably because the system is based on a Japanese MSX2+), while the format says it should be format 1, M/D/Y (06/01/21). (And I would actually expect the format to be 2, D/M/Y (01/06/21))

Compare this to the MSX2+ Japan machine: 
![image](https://user-images.githubusercontent.com/4342/120245430-767c1800-c26d-11eb-804f-431adee36831.png)
https://msxpen.com/codes/-Mb3T6oro5At7F2FCG64
This machine correctly returns 21/06/01 with format 0 (Y/M/D).

The Turbo R Europe machine (as well as the US one) does have matching date and format:
![image](https://user-images.githubusercontent.com/4342/120245532-c064fe00-c26d-11eb-8c9e-d441754bd4d8.png)
https://msxpen.com/codes/-Mb3Ta2Qpzw8AaMsaG6z
(Though again I would expect D/M/Y format 2 for the European one, rather than M/D/Y format 1; but apparently a real Philips NMS 8280 (which I would consider a European machine) also uses the US rather than EU date format...)

**Comment by NullPointerReference** [comment]
Yes; the current WebMSX version patches the #2B (and a few other) parameters, but the PAL/EU/JP ROMs are basically the same ROM per MSX type; see https://github.com/ppeccin/WebMSX/blob/master/src/runtime/sysfiles/EmbeddedSystemFiles.js
and apparently the ROMs only contain code for the format they 'should' specify in #2B.  

I assume that a more extensive diff-patch for these ROMs could fix this without having to include multiple 'full' ROMs.
